app/models/credence_game_response.rb

```diff
@@ -0,0 +1,70 @@
+class CredenceGameResponse < ActiveRecord::Base
+  belongs_to :credence_game
+  belongs_to :credence_question
+  belongs_to :first_answer, class_name: 'CredenceAnswer'
+  belongs_to :second_answer, class_name: 'CredenceAnswer'
+
+  def self.pick_random
+    enabled_questions = CredenceQuestion.where(enabled: true)
+    num_enabled = enabled_questions.count
+
+    question = enabled_questions.first(offset: rand(num_enabled))
+    while !question.enabled || question.weight < rand
+      question = enabled_questions.first(offset: rand(num_enabled))
+    end
+
+    question.create_random_question
+  end
+
+  def text
+    self.credence_question.text
+  end
+
+  def answers
+    [ self.first_answer, self.second_answer ]
+  end
+
+  def answer_correct?(answer)
+    answer == self.correct_index
+  end
+
+  # Check whether answer is the correct answer, and scores according to
+  # credence. credence is a _percentage_, i.e. in the range [1, 99].
+  def score_answer(answer, credence)
+    # A certain-but-wrong answer gives an error anyway, but a certain-and-right
+    # answer just scores 100. We have to reject both, or players can guess with
+    # no consequences.
+    if credence < 1 or credence > 99
+      raise ArgumentError, 'Credence must be between 1 and 99'
+    end
+
+    correct = self.answer_correct? answer
+    truth_credence = correct ? credence : 100 - credence
+    score = (Math.log(2* truth_credence.to_f/100.0, 2) * 100).round
+    return correct, score
+  end
+
+  def score
+    _, s = self.score_answer(self.given_answer, self.answer_credence)
```

Comment:
How do you feel about the _ here? In my lexicon it's the standard name for a variable that never gets used, but I don't know if that's common in ruby.